src/placeos-rest-api/controllers/root.cr

```diff
@@ -74,7 +74,7 @@ module PlaceOS::Api
       render json: Root.construct_versions
     end
 
-    class_getter scopes = SCOPES
+    class_getter(scopes) { SCOPES }
```

Comment:
I think a normal class getter here is correct. 
Generally, lazy getters are good for things like resource acquisition or computationally heavier computed values.

Reply:
The array is empty when using normal class getter

Reply:
Other option is using `macro finished` would that be better?

Reply:
is it not empty otherwise??

Reply:
Might be worth adding a comment as to why its a lazy getter. Someone could make a change and break things if they're not warned here

Reply:
Please add a `NOTE` here noting why the lazy getter is used. I can see someone down the line transforming it in favour of a normal getter and having a WTF moment